### PR TITLE
Fix for kernels selector sed command on macOS

### PR DESCRIPTION
**Pull request by Alexsandruss**
Remove '\b' symbol from sed command on macOS to fix selection of kernels.

### CODE REVIEW

File: .ci/pipeline/ci.yml [hunk 1]
```diff
@@ -382,10 +382,10 @@ jobs:
       .ci/scripts/describe_system.sh
     displayName: 'System info'
   - script: |
-      .ci/scripts/build.sh --compiler clang --target daal --optimizations "sse2 sse42 avx2 avx512" --conda-env ci-env
```
Comment (ahuber21) [inline .ci/pipeline/ci.yml:385]: I think it would be cleaner to have this in a separate commit. Why was it disabled?

Comment (Alexsandruss) [reply .ci/pipeline/ci.yml:385]: @ahuber21 
Previously, subset of ISAs didn't work correctly for building on macOS and all of them were selected in CI. Now, it's changed to SSE2+AVX2

**Review by ahuber21 — APPROVED** [review]

**Review by ahuber21 — APPROVED** [review]

**Review by napetrov — APPROVED** [review]
Assuming CI would pass

**Review by KulikovNikita — APPROVED** [review]

**Comment by Alexsandruss** [comment]
http://intel-ci.intel.com/edf8b235-98b2-f170-8462-a4bf010d0e2e

**Comment by napetrov** [comment]
@homksei  - internal ci preview also can drop number of build targets for macOS
